SDL/SDLMain.cpp

```diff
@@ -1435,6 +1443,33 @@ int main(int argc, char *argv[]) {
 			set_ipad = true;
 		else if (!strcmp(argv[i],"--portrait"))
 			portrait = true;
+		else if (!strncmp(argv[i],"--graphics=",strlen("--graphics="))) {
+			const char *restOfOption = argv[i] + strlen("--graphics=");
+			double val=-1.0; // Yes, floating point.
+			if (!strcmp(restOfOption,"vulkan")) {
```

Comment:
minor style stuff: space after comma in parameter lists. Also after if (which you've done here, but not above). else if goes on the same line as the preceding brace if there is one.